src/test/kotlin/uk/gov/dluhc/printapi/rest/GenerateTemporaryCertificateIntegrationTest.kt

```diff
@@ -253,6 +255,7 @@ internal class GenerateTemporaryCertificateIntegrationTest : IntegrationTest() {
             .isEqualTo("temporary-certificate-${temporaryCertificate.certificateNumber}.pdf")
 
         val pdfContent = response.responseBody
+        FileOutputStream(File("Example_TEMPORARY_CERTIFICATE_English.pdf")).use { it.write(pdfContent) }
```

Comment:
Did you mean to leave this in?

Reply:
No - I forgot to remove it ... I knew there was something and then failed to spot it.

Reply:
Done.